autoload/neomake.vim

```diff
@@ -33,7 +33,11 @@ endfunction
 
 function! neomake#CancelJob(job_id) abort
     if has_key(s:jobs, a:job_id)
-        call jobstop(a:job_id)
+        if has('nvim')
+            call jobstop(a:job_id)
+        else
+            call job_stop(s:jobs[a:job_id].job)
+        endif
```

Comment:
Needs some doc adjustment IIRC (because it refers to `jobstop` explicitly).  Please remove that implementation detail then.